client/src/store/modules/posts.js

```diff
@@ -4,31 +4,13 @@ import axios from 'axios';
  * Define Actions
  * =============================================================
  */
-export const SET_NEW_POST_TITLE_INPUT = 'posts/SET_NEW_POST_TITLE_INPUT';
-export const SET_NEW_POST_CONTENT_TEXTAREA = 'posts/SET_NEW_POST_CONTENT_TEXTAREA';
-export const SET_NEW_POST_LOCATION_INPUT = 'posts/SET_NEW_POST_LOCATION_INPUT';
 
 /** ============================================================
  * Define Initial State
  * =============================================================
  */
 const initialState = {
-  posts: [
-    {
-      title: 'First Post',
-      content: 'This is first post content'
-    },
-    {
-      title: 'Second Post',
-      content: 'This is second post content'
-    },
-  ],
-  newPost: {
-    title: '',
-    content: '',
-    location: ''
-
-  }
+  posts: __PRELOADED_STATE__.posts,
```

Comment:
So all the posts are by a certain user?? But not all the posts from that area?

Reply:
Yup, the current preloaded state is per a single user for now.  Once we implement the distance we can refactor to allow all posts.